src/main/java/com/lpvs/service/GitHubService.java

```diff
@@ -32,12 +39,32 @@ public class GitHubService {
     private String GITHUB_AUTH_TOKEN;
     private String GITHUB_API_URL;
 
-    public GitHubService(@Value("${github.login}") String GITHUB_LOGIN,
-                         @Value("${github.token}") String GITHUB_AUTH_TOKEN,
-                         @Value("${github.api.url:}") String GITHUB_API_URL) {
-        this.GITHUB_LOGIN = GITHUB_LOGIN;
-        this.GITHUB_AUTH_TOKEN = GITHUB_AUTH_TOKEN;
-        this.GITHUB_API_URL = GITHUB_API_URL;
+    private final static String GITHUB_LOGIN_PROP_NAME = "github.login";
+    private final static String GITHUB_AUTH_TOKEN_PROP_NAME = "github.token";
+    private final static String GITHUB_API_URL_PROP_NAME = "github.api.url";
+
+    private final static String GITHUB_LOGIN_ENV_VAR_NAME = "LPVS_GITHUB_LOGIN";
+    private final static String GITHUB_AUTH_TOKEN_ENV_VAR_NAME = "LPVS_GITHUB_TOKEN";
```

Comment:
@v-kerimov 
Please test the Docker case.
When I run the command:
```
docker run -p 7896:7896 --name lpvs -e LPVS_GITHUB_TOKEN="uiuyhggjhg77676ghhg" lpvs:latest
```
the log is:
```
2022-09-09 11:50:24.675  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : Starting LicensePreValidationSystem v1.0.0 using Java 11.0.16 on 475de6038553 with PID 1 (/LPVS/target/lpvs-1.0.0.jar started by root in /LPVS/target)
2022-09-09 11:50:24.680  INFO 1 --- [           main] com.lpvs.LicensePreValidationSystem      : No active profile set, falling back to 1 default profile: "default"
2022-09-09 11:50:28.088  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 7896 (http)
2022-09-09 11:50:28.131  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2022-09-09 11:50:28.133  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.65]
2022-09-09 11:50:28.404  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2022-09-09 11:50:28.404  INFO 1 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 3519 ms
2022-09-09 11:50:28.660 ERROR 1 --- [           main] com.lpvs.service.GitHubService           : LPVS_GITHUB_TOKEN(github.token) is not set.
```